jojo.py

```diff
@@ -107,6 +107,172 @@ def open_json(json_file):
     return df
 
 
+def summon_stand_user_info(mcr):
+    '''
+    スタンド能力と使用者を紐づけるアマスタを生成します。
+    外部ファイルの設定を優先とし、既に生成されている場合はスキップします。
+
+    Parameter
+        mcr : MCRcon
+            Rconのサーバ情報
+    '''
+    str_dir = 'json_list'
+    str_stand_file = 'stand_list.json'
+    entity_name = 'minecraft:armor_stand'
+    X = 0
+    Y = -74
+    Z = 0
+    invulnerable = True
+    gravity = False
+    contnts = open_json(f'{str_dir}/{str_stand_file}')
+
+    for key in contnts.keys():

```

Comment:
keyが何か分かりにくいのでこの際standの名前にしましょう。

for key in contnts.keys():
```suggestion
    for stand_name in contnts.keys():

```

Reply:
keyをstand_nameに修正しました。